tsconfig.json

```diff
@@ -1,8 +1,15 @@
 {
   "compilerOptions": {
-    "target": "es3",
+    "forceConsistentCasingInFileNames": true,
+    "incremental": true,
+    "isolatedModules": true,
     "moduleResolution": "node",
-    "outDir": "./dist"
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": false,
```

Comment:
Ideally we should remove this line, but it is *a lot* of work to type _everything_ 😅 
```suggestion
```

Reply:
Update: this line is removed in #11  :tada: